docs/paas-infrastructure.md

```diff
@@ -0,0 +1,21 @@
+# PaaS Infrastructure 
+
+## Purpose
+
+This document describes the PaaS ([GOV.UK Platform as a Service](https://www.cloud.service.gov.uk)) architecture that the Apply for Teacher Training service is deployed into.
+
+## PaaS spaces
+
+The apply service is deployed across three PaaS 'spaces' that are shared with other services in BAT.
+
+- `apply-prod` contains the *production* and *sandbox* Apply environment.
+- `apply-staging` contains the *staging* Apply environment.
+- `apply-qa` contains the *qa* Apply environment.
+
+## Resource permissions
```

Comment:
For Azure this section described PIM. Is there anything we need to say about PaaS and permissions here?

Reply:
We're planning to discuss if we want to follow something similar to PIMs vs give everyone access as it stands, hopefully in the coming week, I think for now we can leave this out. Another section I can think of are secrets in Azure, do we want to add a section around those?